3rdParty/csv-parser/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 3.15)
+project(csvparser)
+option (CSVTEST "create csv test program " OFF)
+
+
+add_library(csvparser STATIC csv_parser.cpp)
+target_include_directories(csvparser PUBLIC .)
+
+if (CSVTEST)
+    add_executable(csvparser_test csv_parser.cpp test.cpp)
+endif()
+
+
```

Comment:
it looks like you deleted his 3rd party links

Reply:
I did delete the links from clicking on the folder. I didn't know how else to get CMakeLists.txt in the folder, because I couldn't commit those folders while they were links. the README still has the links in them though. But I would be happy to do something else if you think it would be better.

Reply:
What?